### ISSUE TITLE
Two page loads needed for offline support to work

### ISSUE THREAD

**Issue by ile** [sveltejs/sapper#433]
I first created this issue here: https://github.com/sveltejs/svelte/issues/1583

The problem: the offline support (with service workers) seem to work only after two loads of the same webpage. That is, the initial load + a refresh.

This happens at least with the default Sapper template (Borat).

This can be tested easily with the Chrome devtools/offline checkbox.

**Comment by laurentpayot** [comment]
Is it the same issue as https://github.com/sveltejs/sapper/issues/422 ?

**Comment by mwilliams1188** [comment]
This may be unrelated, but I've been going nuts over an issue the last few days where all my pages are doing this double load thing. The page loads, runs scripts, then refreshes (and undoes any scripts I ran).

Could this be related to offline/cache stuff? I'm pretty new to sapper and I haven't changed any of the default config, just added components and css. I'll probably post this as a separate issue in a bit but just wanted to throw it here. Thanks
